### DISCUSSION TITLE
Set a page as homepage in WordPress

### DISCUSSION THREAD

**Discussion by jenishshrestha** [vercel/next.js#23030] [Help]
I am new to next js.

I have trying to integrate WordPress with next js and so far its going well.
I used getStaticPaths to catch all the routes of pages and the pages that I created in WordPress backend is showing up correctly.

Now, I create a new page in WordPress backend and set it as a homepage, next js fail to generate it as a homepage in Next js frontend.

How do I make this work?

**Comment by Emiliano-Bucci** [comment]
@jenishshrestha Hi! Did you find any solution to this?
